src/app/static/.eslintrc.js

```diff
@@ -1,29 +1,30 @@
 module.exports = {
-  root: true,
-  env: {
-    node: true,
-  },
-  extends: [
-    'plugin:vue/vue3-essential',
-    '@vue/airbnb',
-    '@vue/typescript/recommended',
-  ],
-  parserOptions: {
-    ecmaVersion: 2020,
-  },
-  rules: {
-    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
-  },
-  overrides: [
-    {
-      files: [
-        '**/__tests__/*.{j,t}s?(x)',
-        '**/tests/unit/**/*.spec.{j,t}s?(x)',
-      ],
-      env: {
-        jest: true,
-      },
+    root: true,
+    env: {
+        node: true
     },
-  ],
+    extends: [
+        "plugin:vue/vue3-essential",
+        "@vue/airbnb",
+        "@vue/typescript/recommended"
+    ],
+    parserOptions: {
+        ecmaVersion: 2020
+    },
+    rules: {
+        "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
+        "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
+        "comma-dangle": ["error", "never"],
```

Comment:
I don't know if this merits an exception - dangling commas seem to be a harmless way of minimising diff sizes. But I'm easy (and I don't really want to waste time reverting!).

Reply:
Maybe it doesn't, I just find them visually confusing/annoying. We can allow them back in if people would prefer them.